raiden-dapp/src/components/AmountInput.vue

```diff
@@ -149,40 +186,33 @@ export default class AmountInput extends Vue {
 @import '../scss/mixins';
 @import '../scss/colors';
 @import '../scss/fonts';
-
```

Comment:
Formatting issues here? :thinking:

Reply:
Ah, sorry, thought this got taken care of yesterday. I pushed changes now.